README.md

```diff
@@ -62,6 +63,36 @@ karma start
 
 Specs live under the `spec` directory and can be configured by editing the `karma.conf.js` file.
 
+## Deploying
+
+To check that your app will pass the server-side validation check, run
+
+```
+zat validate --path=./dist
+```
+
+If validation is successful, you can upload the app into your Zendesk account by running
```

Comment:
In this case I actually think Zendesk is correct. All apps are uploaded through the Zendesk account, which shared between products.